ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_logic.py

```diff
@@ -1143,3 +1143,56 @@ def test_jax_numpy_packbits(
         axis=axis,
         bitorder=bitorder,
     )
+
+
+@st.composite
+def _func_and_shape_dtype_helper(draw):
+    # here assumption is that the input func will take the len(shape) no of parameters
+    def add_numbers(*args):
+        total = 0
+        for num in args:
+            total += num
+        return total
+
+    shape = draw(
+        helpers.get_shape(
+            allow_none=False,
+            min_num_dims=1,
+            max_num_dims=1,
+            min_dim_size=1,
+            max_dim_size=1,
+        )
+    )
+
+    dtype = draw(st.sampled_from(["int64", "float64", "float32", "int32", "uint8"]))
```

Comment:
this dtype is not used anywhere. better to remove!

Reply:
actually, dtype is being used, it is one of the parameter of fromfunction